core/src/main/java/com/linecorp/armeria/internal/common/util/ChannelUtil.java

```diff
@@ -62,6 +72,43 @@
         PROHIBITED_OPTIONS = builder.build();
     }
 
+    @Nullable private static ChannelOption<?> epollTcpUserTimeout;
+    @Nullable private static ChannelOption<?> epollTcpKeepidle;
+    @Nullable private static ChannelOption<?> epollTcpKeepintvl;
+    @Nullable private static ChannelOption<?> ioUringTcpUserTimeout;
+    @Nullable private static ChannelOption<?> ioUringTcpKeepidle;
+    @Nullable private static ChannelOption<?> ioUringTcpKeepintvl;
+
+    static {
+        try {
+            final Class<?> clazz = Class.forName(
+                    "io.netty.channel.epoll.EpollChannelOption", false,
+                    ChannelUtil.class.getClassLoader());
+            epollTcpUserTimeout = findChannelOption(clazz, "TCP_USER_TIMEOUT");
+            epollTcpKeepidle = findChannelOption(clazz, "TCP_KEEPIDLE");
+            epollTcpKeepintvl = findChannelOption(clazz, "TCP_KEEPINTVL");
+        } catch (Throwable throwable) {
+            // Ignore
+        }
+        try {
+            final Class<?> clazz = Class.forName(
+                    "io.netty.incubator.channel.uring.IOUringChannelOption", false,
+                    ChannelUtil.class.getClassLoader());
+            ioUringTcpUserTimeout = findChannelOption(clazz, "TCP_USER_TIMEOUT");
+            ioUringTcpKeepidle = findChannelOption(clazz, "TCP_KEEPIDLE");
+            ioUringTcpKeepintvl = findChannelOption(clazz, "TCP_KEEPINTVL");
+        } catch (Throwable throwable) {
+            // Ignore
+        }
```

Comment:
How about moving `try...catch` block into `findChannelOption` to proceed to the next fields even if a field is failed to find.